rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/terminal/ITerminalIDEServer.java

```diff
@@ -233,6 +232,30 @@ public String toString() {
         }
     }
 
+    public static class EditorParameter {
```

Comment:
shouldn't this extend `ShowDocumentParams`? to denote it's an extention of lsp feature?

Reply:
Also it's not extending showDocument, it's replacing it. So documentation of the old feature will not remain relevant.  It's better (more stable) to let them evolve independently.